docs/add-ons/configuration.md

```diff
@@ -184,7 +184,7 @@ image: repo/{arch}-my-custom-addon
 | init | bool | no | Default `true`. Disable the Docker default system init.  Use this if the image has its own init system.
 | watchdog | string | no | An URL for monitor an application this add-on. Like `http://[HOST]:[PORT:2839]/dashboard`, the port needs the internal port, which will be replaced with the effective port. It is also possible to bind the protocol part to a configuration options with: `[PROTO:option_name]://[HOST]:[PORT:2839]/dashboard` and it's looked up if it is `true` and it's going to `https`. For simple TCP port monitoring you can use `tcp://[HOST]:[PORT:80]`. It work for add-ons on host or internal network.
 | realtime | bool | no | Give add-on access to host schedule including SYS_NICE for change execution time/priority.
-| journald | bool | no | Default `false`. If set to `true`, the host's journald will be mapped read-only into the add-on at `/var/logs/journal`
+| journald | bool | no | Default `false`. If set to `true`, the host's system journal will be mapped read-only into the add-on. It will either be available at `/var/log/journal` or `/run/log/journal` depending on the build of your addon.
```

Comment:
The volatile is handled by the Supervisor and mounted in the persistent location inside the add-on. For the add-on maintainer, there is not need to know the difference because of that.

```suggestion
| journald | bool | no | Default `false`. If set to `true`, the host's system journal will be mapped read-only into the add-on at `/var/log/journal`
```

Reply:
@frenck Was that the intent of https://github.com/home-assistant/supervisor/pull/2742 ? Because that's not actually what I'm seeing. I'm on `2021.03.8` and here's what I see in the SSH & Web Terminal addon (as well as my Promtail one):
<img width="835" alt="Screen Shot 2021-03-27 at 9 44 26 PM" src="https://user-images.githubusercontent.com/2037026/112739832-a34a4100-8f45-11eb-92ac-1e146247909e.png">

Looking at https://github.com/home-assistant/supervisor/blob/main/supervisor/docker/addon.py#L427 if that was the intent I think the source and target may have been flipped. It's adding `/var/log/journal` as a new key to `volumes` every time and changing the `bind` target based on whether or not the persistent journal exists.

Sorry I would've said something sooner, I thought that PR was saying there was a problem with the implementation in containers built on debian, I didn't realize it was an issue with hosts.

Reply:
Yes, its a bug. I even wrote tests to verify the bug as not a bug. Awesome! 😢 

Will fix it.

Reply:
Ah ok. No worries, happens to the best of us.

I fixed this PR, no more mention of `/run/log/journal`.